taprpc/tapdevrpc/tapdev.swagger.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
I think we should squash this commit with the previous one, otherwise the first one won't compile?